src/api/dependency-discovery/src/dependency-list.js

```diff
@@ -46,6 +46,15 @@ async function getNpmPackageInfo(packageName) {
 
   if (dependencies[packageName] === null) {
     const { id, license, gitRepository } = await getPackument({ name: packageName });
+
+    // Create Github issue URL that filter open issue with labels hacktoberfest, good first issue, or help wanted.
+    if (gitRepository?.url) {
+      const labels = ['"hacktoberfest"', '"good first issue"', '"help wanted"'];
```

Comment:
Could you make it so that the labels defined here and [there](https://github.com/Seneca-CDOT/telescope/blob/7973b2d7d9253228a705c29d0d866919fd608051/src/api/dependency-discovery/src/github.js#L11) is instead a global variable exported in `github.js`, so that we don't have two places with the same set of labels?